### ISSUE TITLE
Unhandled UI Exception

### ISSUE THREAD

**Issue by wadtech** [EDDiscovery/EDDiscovery#1487]
### Description
Loading the application.
Splash screens shows 'Modulating Shields'

### Additional Information
EDDiscovery 8.4.1.0

### Exception Details:
>There was an unhandled UI exception.
>```
>==== BEGIN ====
>System.InvalidOperationException: This operation cannot be performed while an auto-filled column is being resized.
>   at System.Windows.Forms.DataGridView.PerformLayoutPrivate(Boolean useRowShortcut, Boolean computeVisibleRows, Boolean invalidInAdjustFillingColumns, Boolean repositionEditingControl)
>   at System.Windows.Forms.DataGridView.SetColumnHeadersHeightInternal(Int32 columnHeadersHeight, Boolean invalidInAdjustFillingColumns)
>   at System.Windows.Forms.DataGridView.AutoResizeColumnHeadersHeight(Boolean fixedRowHeadersWidth, Boolean fixedColumnsWidth)
>   at System.Windows.Forms.DataGridView.OnColumnHeadersGlobalAutoSize()
>   at System.Windows.Forms.DataGridView.set_TopLeftHeaderCell(DataGridViewHeaderCell value)
>   at System.Windows.Forms.DataGridView.get_TopLeftHeaderCell()
>   at System.Windows.Forms.DataGridView.OnCellMouseEnter(DataGridViewCellEventArgs e)
>   at System.Windows.Forms.DataGridView.UpdateMouseEnteredCell(HitTestInfo hti, MouseEventArgs e)
>   at System.Windows.Forms.DataGridView.OnColumnWidthChanged(DataGridViewColumnEventArgs e)
>   at System.Windows.Forms.DataGridView.AdjustFillingColumns()
>   at System.Windows.Forms.DataGridView.ComputeLayout()
>   at System.Windows.Forms.DataGridView.PerformLayoutPrivate(Boolean useRowShortcut, Boolean computeVisibleRows, Boolean invalidInAdjustFillingColumns, Boolean repositionEditingControl)
>   at System.Windows.Forms.DataGridView.OnHandleCreated(EventArgs e)
>   at System.Windows.Forms.Control.WmCreate(Message& m)
>   at System.Windows.Forms.Control.WndProc(Message& m)
>   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
>===== END =====
>```

**Comment by phroggster** [comment]
`Controller.Init();` is responsible for a lot, but I'm not sure what we could be doing wrong inside of it. The lack of EDDiscovery stack frames in that trace make for quite a challenge to diagnose.

**Comment by klightspeed** [comment]
```
  at System.Windows.Forms.DataGridView.OnCellMouseEnter(DataGridViewCellEventArgs e)
  at System.Windows.Forms.DataGridView.UpdateMouseEnteredCell(HitTestInfo hti, MouseEventArgs e)  <--
  at System.Windows.Forms.DataGridView.OnColumnWidthChanged(DataGridViewColumnEventArgs e)
  at System.Windows.Forms.DataGridView.AdjustFillingColumns()
```

This is odd, as is the nested `System.Windows.Forms.DataGridView.PerformLayoutPrivate`

**Comment by klightspeed** [comment]
Same issue as in <https://stackoverflow.com/questions/34344499/invalidoperationexception-this-operation-cannot-be-performed-while-an-auto-fill>

It's still present in 4.7.1 (the test case in that thread gives the same exception), and according to <http://connect.microsoft.com/VisualStudio/feedback/details/481029/this-operation-cannot-be-performed-while-an-auto-filled-column-is-being-resized> it's "by design".

Exception from test case:
```
   at System.Windows.Forms.DataGridView.PerformLayoutPrivate(Boolean useRowShortcut, Boolean computeVisibleRows, Boolean invalidInAdjustFillingColumns, Boolean repositionEditingControl)
   at System.Windows.Forms.DataGridView.SetColumnHeadersHeightInternal(Int32 columnHeadersHeight, Boolean invalidInAdjustFillingColumns)
   at System.Windows.Forms.DataGridView.AutoResizeColumnHeadersHeight(Boolean fixedRowHeadersWidth, Boolean fixedColumnsWidth)
   at System.Windows.Forms.DataGridView.OnColumnHeadersGlobalAutoSize()
   at System.Windows.Forms.DataGridView.set_TopLeftHeaderCell(DataGridViewHeaderCell value)
   at System.Windows.Forms.DataGridView.get_TopLeftHeaderCell()
   at System.Windows.Forms.DataGridView.GetCellInternal(Int32 columnIndex, Int32 rowIndex)
   at System.Windows.Forms.DataGridView.OnCellMouseEnter(DataGridViewCellEventArgs e)
   at System.Windows.Forms.DataGridView.UpdateMouseEnteredCell(HitTestInfo hti, MouseEventArgs e)
   at System.Windows.Forms.DataGridView.OnColumnWidthChanged(DataGridViewColumnEventArgs e)
   at System.Windows.Forms.DataGridView.OnBandThicknessChanged(DataGridViewBand dataGridViewBand)
   at System.Windows.Forms.DataGridViewBand.set_ThicknessInternal(Int32 value)
   at System.Windows.Forms.DataGridView.AdjustFillingColumns()
   at System.Windows.Forms.DataGridView.ComputeLayout()
   at System.Windows.Forms.DataGridView.PerformLayoutPrivate(Boolean useRowShortcut, Boolean computeVisibleRows, Boolean invalidInAdjustFillingColumns, Boolean repositionEditingControl)
   at System.Windows.Forms.DataGridView.OnHandleCreated(EventArgs e)
   at System.Windows.Forms.Control.WmCreate(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.DataGridView.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by klightspeed** [comment]
Replicated by putting the mouse cursor at (18,147) on the screen and running the program.

#1490 (master) and #1491 (v8.4.x) should fix this.

**Comment by klightspeed** [comment]
This should be fixed in 8.4.2

**Comment by wadtech** [comment]
Thanks for looking at my issue so quickly. I appreciate your hard work :-)
